src/app/learn/get-started/prereqs/ssl-certificate/index.html

```diff
@@ -76,6 +76,11 @@
         In the prompt that appears, select <strong>Allow</strong> to open a settings page.
```

Comment:
This no longer opens the settings page, instead, it shows an alert that says "Profile Downloaded. Review the profile in Settings app if you want to install it.", and the only option is "Close".

So this line could say "... select <strong>Allow</strong> and then <strong>Close</strong>" and then I think it's good!